### PR TITLE
initial spec

### PR DESCRIPTION
**Pull request by beausmith**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,67 @@
...
+2. Admin can edit the election configuration via simple UI forms at any time.
+3. Admin can list all ballots by precinct/style.
+4. Admin can view an individual ballot, both HTML and PDF versions.
+5. Admin can export an election ballot package for [Mail Ballot Manager](https://github.com/votingworks/mail-ballot-manager).
+
+   Election Ballot Package contains:
+     - `election.json` - VotingWorks data format for election data: contests, canidates,ballot styles, precincts, etc.
+     - approved official ballots in pdf format:
+       - one for each combination of ballot style and precinct
+       - file name pattern: `ballot-style-7-precinct-8.pdf`
```
Comment (benadida) [inline README.md:22]: can we build an election ID into the filename too?

File: README.md [hunk 2]
```diff
@@ -1,2 +1,67 @@
...
+- Election Editor with sections to edit, metadata, parties, contests, ballot styles, etc.
+  - Edit Contest section
+    - List Contests screen
+    - Edit Contest Screen with live preview
+
+## Open Questions
+- Do we implement anything to ensure that ballots are not edited after package is created?
+  - possibly a `ballots.json` with PDF ballot file hashes?
+  - possibly a hash of the package as a separate file?
+
```
Comment (benadida) [inline README.md:66]: we could have a manifest of ballot files with hashes, but I think best to start simple with a hash of the election package displayed at creation time and at load time in other components.

File: README.md (hunk 1 above)
Comment (beausmith) [reply README.md:22]: Would this election id be a top-level key in the `election.json` file or something generated from other data in the election configuration?

Comment (benadida) [reply README.md:22]: it could be a hash of `election.json`, which would be long, but would certainly prevent any confusion.

**Pull request merged**